docs: add mentoring guide (#105)
thoth-tech/handbook: docs/communication/mentoring-guide.md

# Mentoring Guide

## Why is mentoring important?

> - Aligns with the values set out in the Thoth Tech company mission and chartering
> - Supporting professional and personal development
> - Helps achieve company goals
> - Empowering growth in others
> - Mentors serve as a source of knowledge and support
> - Helping set goals
> - Helping keep mentees accountable to their goals
> - Helping motivate and encourage to persist through challenges
> - Creating connections and networking
> - Creating trust and rapport
> - Offering safe honest fearless feedback
> - Strengthens knowledge
> - Help you gain new perspectives
> - Provides a sense of fulfillment

Reviewer: **suggestion:** It might look more uniformly to use infinitive verbs when applies. WDYT?

```suggestion
- Align with the values set out in the Thoth Tech company mission and chartering
- Support professional and personal development
- Help achieve company goals
- Empower growth in others
- Mentors serve as a source of knowledge and support
- Help set goals
- Help keep mentees accountable to their goals
- Help motivate and encourage to persist through challenges
- Create connections and networking
- Create trust and rapport
- Offer safe honest fearless feedback
- Strengthen knowledge
- Help you gain new perspectives
- Provide a sense of fulfilment
```

## Key Mentoring Skills

> - Communication (see guide on [Fearless Feedback](docs/../fearless-feedback.md))

Reviewer: **suggestion:** We can use a relative path.

```suggestion
- Communication (see guide on [Fearless Feedback](fearless-feedback.md))
```

> - Interpersonal saavy

Reviewer: ```suggestion
- Interpersonal savvy
```

- Leadership
- Confidence
- Understanding resources
- Identifying Goals
- Active Listening
- Asking permission
- Empathy

## Mentoring models

### One on One Mentoring

- Regular check ins
- One on one tutorials or conversations

### Peer Mentoring

- Empowering others to share their knowledge
- Partnering up Seniors with Juniors as buddies
- Partnering up someone skilled in an area to support someone less skilled needing help

### Group mentoring

- Workshops
- Tutorials
- How-to guides

### SMART goal model

Defining goals with these parameters can help ensure that your objectives are realistic and
attainable within a certain time-frame such as the Capstone project.

> **S**pecific **M**easurable **A**cheivable **R**elevant **T**imebound

Reviewer: ```suggestion
**S**pecific **M**easurable **A**chievable **R**elevant **T**imebound
```

## Guide to being a Successful Mentor

1. Understand what you want out of the relationship.
   - Do you view the relationship as bi-directional where you learn from them as much as they learn
     from you?

>    - How does taking on a mentorship role strangthen you as a leader in your personal and

Reviewer: ```suggestion
   - How does taking on a mentorship role strengthen you as a leader in your personal and
```

     professional life?
2. Set expectations together in the very beginning.

>    - How often do you have availibility?

Reviewer: ```suggestion
   - How often do you have availability?
```

   - How hands-on can you be yourself?
   - What resources can you provide?
   - Is there a time limit on the relationship?
3. Take a genuine interest in your mentee as a person.
   - Get to know your mentee on a deeper level will how you build and strong relationship, and
     understanding them as person and how they interact with others.
4. Build trust and rapport.

>    - Vulnerability is key to a successful mentoship. Fostering a safe environment through trust and

Reviewer: ```suggestion
   - Vulnerability is key to a successful mentorship. Fostering a safe environment through trust and
```

     rapport will help you assess core issues, challenges and opportunities beyond circumstance.
5. Know when to give advice.
   - Not all feedback is helpful feedback, learning to know the difference is important.
   - Sometimes taking some time to ensure you have the right information, experience or emotional
     state will help you come back with a clear and valuable response.
6. Don’t assume anything about your mentee – ask.
   - Everyone has bias.
   - Bias clouds our judgement whether you are conscious of it or not.
   - Breakthrough common assumptions by asking questions and digging deeper.
7. Share your journey.
   - Be open to being vulnerable and sharing your own mistakes and failures.

>    - This builds trust and relatibilty.

Reviewer: ```suggestion
   - This builds trust and relatability.
```

8. Celebrate their achievements.
   - taking the time to highlight and celebrate successes and achievements helps build a mentee's
     confidence and keeps them motivated
9. Seek out resources to help your mentee grow.

>    - Greate mentors look out for or create situations to help their mentees get closer to their

Reviewer: ```suggestion
   - Great mentors look out for or create situations to help their mentees get closer to their
```
